benchmark/kvbench/test/sequential_custom_traffic_perftest.py

```
@@ -141,6 +614,441 @@ def _get_bufs(self, tp: TrafficPattern):
 
         return send_bufs, recv_bufs
 
+    # =========================================================================
+    # WARMUP AND BENCHMARK HELPERS
+    # =========================================================================
+
+    def _run_warmup(self, tp_handles, storage_read_handles, storage_write_handles):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_run_warmup`

Add return type annotation: `None`

(ANN202)